pkg/arm/data/master-startup.sh

```diff
@@ -44,9 +44,7 @@ fi
 
 echo "BOOTSTRAP_CONFIG_NAME=node-config-master" >>/etc/sysconfig/${SERVICE_TYPE}-node
 
-{{- if $.Extra.TestConfig.RunningUnderTest }}
-sed -i -e "s#DEBUG_LOGLEVEL=2#DEBUG_LOGLEVEL=4#" /etc/sysconfig/${SERVICE_TYPE}-node
-{{- end }}
+sed -i -e "s#DEBUG_LOGLEVEL=2#DEBUG_LOGLEVEL={{ .Config.ClusterLogLevel }}#" /etc/sysconfig/${SERVICE_TYPE}-node
```

Comment:
I think with these changes we're losing the ability to get debug logs when `RUNNING_UNDER_TEST`, I'd suggest work that back into the dev process, either by defaulting the logLevel to 4 when running under test, or exposing the logLevel to be configurable through environment variables.

Reply:
i can readd this in a follow-up

Reply:
@jim-minter just to make sure, are you ok with @thekad 's suggestion? I think I prefer his second option. what do you think?